.github/workflows/build.yml

```diff
@@ -47,25 +47,25 @@ jobs:
           cd tvb_build
           bash install_full_tvb.sh
 
-      - name: cache data
-        id: cache-data
-        uses: actions/cache@v3
-        with:
-          path: tvb_data
-          key: tvb-data
+      #- name: cache data
+      #  id: cache-data
+      #  uses: actions/cache@v3
+      #  with:
+      #    path: tvb_data
+      #    key: tvb-data
```

Comment:
too much deprecated code will be left behind in commented lines.
Please consider to now remove the old unnecessary code.
git history can help us with previous versions

Reply:
Yes right! Will remove the commented lines